scio-snowflake/src/main/scala/com/spotify/scio/snowflake/SnowflakeOptions.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2024 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.snowflake
+
+trait SnowflakeAuthenticationOptions
```

Comment:
```suggestion
sealed trait SnowflakeAuthenticationOptions
```
To avoid the `match may not be exhaustive` warning

Reply:
TIL
fixup 25c5a76e3d4e203b7054ab4a45a92be6d901e940

Reply:
fixup 9ede3f34fda4f1f2c35f387aee9dffc8a4bce3b7